framework/src/wifi_manager/wifi_manager_lwnl_listener.c

```diff
@@ -97,7 +98,11 @@ static inline void LWNL_SET_MSG(wifimgr_msg_s *msg, wifimgr_evt_e event,
 
 static int _lwnl_call_event(int fd, lwnl_cb_status status, int len)
 {
-	switch (status) {
+	if (status.type != LWNL_DEV_WIFI) {
+		return 0;
```

Comment:
In the below, failure case returns `-1`. Is it not the failure case?

Reply:
it's not failure case.